src/components/attribution/AttributionDirective.js

```diff
@@ -1,22 +1,27 @@
 goog.provide('ga_attribution_directive');
+
+goog.require('ga_browsersniffer_service');
+goog.require('ga_debounce_service');
+goog.require('ga_map_service');
+
 (function() {
 
-  var module = angular.module('ga_attribution_directive', []);
+  var module = angular.module('ga_attribution_directive', [
+    'ga_browsersniffer_service',
+    'ga_map_service',
+    'ga_debounce_service'
```

Comment:
Same: I'd have expected to see `'ga_attribution_service'` here given that `gaAttribution` is injected into the directive factory.